tests/cli/test_repos_sanity.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+# Note: execute this file from the project root directory
+
+set -e
+
+. /usr/share/beakerlib/beakerlib.sh
+. $(dirname $0)/lib/lib.sh
+
+CLI="${CLI:-./src/bin/composer-cli}"
+
+rlJournalStart
+    rlPhaseStartSetup
+        repodir_backup=$(mktemp -d composerrepos-XXXXX)
+
+        if [[ -z "$CLI" || "$CLI" == "./src/bin/composer-cli" ]]; then
+            composer_params=$(ps ax | grep -v grep  | grep -m 1 -Po 'bin/lorax-composer \K.*')
+        else
+            composer_params="/var/lib/lorax/composer/blueprints"
+        fi
+        composer_stop
+
+        rlRun -t -c "mv /var/lib/lorax/composer/repos.d/* $repodir_backup"
+        rlRun -t -c "mv /etc/yum.repos.d /etc/yum.repos.d.disabled"
+    rlPhaseEnd
+
+    rlPhaseStartTest "Run lorax-composer with --no-system-repos option"
+        composer_start --no-system-repos $composer_params &
+        wait_for_composer
+        # check that there are no composer repos available
+        rlRun -t -c "$CLI sources list | grep -v '^$' | wc -l | grep '^0$'"
+        rlRun -t -c "$CLI compose start example-http-server qcow2 2>&1 | grep -q \
+            'Problem depsolving example-http-server:'"
+        composer_stop
+
+        rlRun -t -c "mv $repodir_backup/* /var/lib/lorax/composer/repos.d"
+        composer_start --no-system-repos $composer_params &
+        wait_for_composer
+        UUID=$(composer-cli compose start example-http-server partitioned-disk)
+        rlAssertEquals "exit code should be zero" $? 0
+        UUID=$(echo $UUID | cut -f 2 -d' ')
+
+        if [ -n "$UUID" ]; then
+            until $CLI compose info $UUID | grep 'FINISHED\|FAILED'; do
+                sleep 10
+                rlLogInfo "Waiting for compose to finish ..."
+            done;
+            check_compose_status "$UUID"
+        else
+            rlFail "Compose UUID is empty!"
+        fi
```

Comment:
I fail to understand what the lines 35-50 are for. From what I can see:

- restore the `/var/lib/lorax/composer/repos.d` directory with its original content, which is all repos from `/etc/yum.repos.d`, which in turn got there when we started lorax-composer (in `test_cli.sh`) before running this script
- then make sure we can build an image which we do in many other places in the test suite so not really needed to do it here.

The only thing in lines 35-50 which I could think off is to try validating that --no-system-repos will not copy files from `/etc/yum.repos.d` which is already done in the snippet above. You may want to verify that existing files are/are not overwritten but that is a completely different test scenario. 

Please explain lines 35-50 to me b/c as they are right now I don't think they are useful.